src/components/Dashboard/Dashboard.tsx

```diff
@@ -108,19 +95,33 @@ export const Dashboard = () => {
         )}
         {formVisible && (
           <form className={style.form} onSubmit={handleSubmit}>
-            <Row fullWidth justifyStart>
-              <label htmlFor="activity">
-                aktivita:
-                <input
-                  id="activity"
+            <Row fullWidth justifyStart flexCol>
+              <TextField
+                type="text"
+                label="aktivita"
+                placeholder="běhání"
+                className={style.input}
+                onChange={(e) => setNewActivity(e.target.value)}
+                autoFocus
+              />
+              <div className={style.inputWrapper}>
+                <TextField
+                  type="number"
+                  label="množství"
+                  placeholder="30"
+                  className={style.input}
+                  onChange={(e) => setNewAmount(+e.target.value)}
+                />
+                <TextField
                   type="text"
-                  placeholder="běhání"
+                  label="jednotka"
+                  placeholder="minut"
```

Comment:
If you want a default value, use defaultValue. Use a placeholder if you want some explanation, additional context, or an example. Be careful what you add in there. It looks like defaultValue right now. Especially with the design, I remember where placeholder and value had the exact same color.

Placeholder's color and value are accessibility matters as well, btw.

Reply:
should be fixed in previous branch, I will create special branch only to fix defaultValues in changeProfile form, I dont think that otherwise the dafaultValue will be needed